docs/about/internals/sync.md

```diff
@@ -50,49 +54,62 @@ Checkpoints:
 5. Begin [sync-superblock protocol](./vsr.md#protocol-sync-superblock).
 6. [Request superblock checkpoint state](#6-request-superblock-checkpoint-state).
 7. Update the superblock headers with:
-    - Bump `vsr_state.checkpoint.header` to the sync target header.
-    - Bump `vsr_state.checkpoint.parent_checkpoint_id` to the checkpoint id that is previous to our sync target (i.e. it isn't _our_ previous checkpoint).
-    - Bump `replica.commit_min`. (If `replica.commit_min` exceeds `replica.op`, transition to `status=recovering_head`).
-    - Set `vsr_state.sync_op_min` to the minimum op which has not been repaired.
-    - Set `vsr_state.sync_op_max` to the maximum op which has not been repaired.
+   - Bump `vsr_state.checkpoint.header` to the sync target header.
+   - Bump `vsr_state.checkpoint.parent_checkpoint_id` to the checkpoint id that is previous to our
+     sync target (i.e. it isn't _our_ previous checkpoint).
+   - Bump `replica.commit_min`. (If `replica.commit_min` exceeds `replica.op`, transition to
+     `status=recovering_head`).
+   - Set `vsr_state.sync_op_min` to the minimum op which has not been repaired.
+   - Set `vsr_state.sync_op_max` to the maximum op which has not been repaired.
 8. Sync-superblock protocol is done.
-9. Repair [replies](./vsr.md#protocol-sync-client-replies), [free set, client sessions, and manifest blocks](./vsr.md#protocol-repair-grid), and [table blocks](./vsr.md#protocol-sync-forest) that were created within the `sync_op_{min,max}` range.
+9. Repair [replies](./vsr.md#protocol-sync-client-replies),
+   [free set, client sessions, and manifest blocks](./vsr.md#protocol-repair-grid), and
+   [table blocks](./vsr.md#protocol-sync-forest) that were created within the `sync_op_{min,max}`
+   range.
 10. Update the superblock with:
     - Set `vsr_state.sync_op_min = 0`
     - Set `vsr_state.sync_op_max = 0`
 
-If a newer sync target is discovered during steps *5*-*6* or *9*, go to step *4*.
+If a newer sync target is discovered during steps _5_-_6_ or _9_, go to step _4_.
```

Comment:
BTW, would be great to document the style we use somewhere, just so that we don't flip back and forth between * and _

Reply:
I'm not sure how folks would feel about introducing a tool that does this but I think it would make sense to use a formatting tool like Prettier, include the config file, and include a step in the build process that checks that it matches the given format

Reply:
This mostly depends on the tool! I think our decision matrix is

```
good tool > no tool > bad tool
```

For prettier in particular, I think it might get disqualified because our long term plan is "npm is banned". We don't want to depend on JS ecosystem as it is tends to be brittle, slow, and churn-prone.

Reply:
That's fair. I looked around and there doesn't seem to be a good option for markdown formatting in a different language that's well supported.

I don't love this option but we could document the style we use + include a prettier config file in the repo, but not enforce its use with a build step...